dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardUnavailableArtifactTest.java

```diff
@@ -118,12 +120,17 @@ public void testDashboardWithRepositoryException()
     validArtifactResult.addResult(DashboardMain.TEST_NAME_UPPER_BOUND, 0);
     validArtifactResult.addResult(DashboardMain.TEST_NAME_DEPENDENCY_CONVERGENCE, 0);
     validArtifactResult.addResult(DashboardMain.TEST_NAME_GLOBAL_UPPER_BOUND, 0);
+    DependencyPath rootValid = new DependencyPath(validArtifact);
+    validArtifactResult.setDependencyTree(ImmutableListMultimap.of(rootValid, rootValid));
```

Comment:
why are we passing the same object as key and value?

Reply:
It's a convention of the multipmap<DependencyPath, dependencyPath>. I'll try to use `DependencyGraph` and this will be gone.

Reply:
This piece of code is removed.